### PR TITLE
Add typed/images/compile-time.rkt

### PR DESCRIPTION
**Pull request by wargrey**
copied from `images/compile-time` with few modifications.

### CODE REVIEW

**Comment by wargrey** [comment]
So just put a tests into typed-racket-more/succeed?

**Comment by samth** [comment]
Yes, exactly.

**Comment by wargrey** [comment]
Okay, it's done.

File: typed-racket-test/succeed/pr476-compile-time-images.rkt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#lang typed/racket
+
+(require "../../typed-racket-more/typed/images/compile-time.rkt")
```
Comment (samth) [inline typed-racket-test/succeed/pr476-compile-time-images.rkt]: I think this should just be `(require typed/images/compile-time)`.

**Pull request merged**